modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -834,6 +820,55 @@ private long _getObjectFolderId(String objectFolderExternalReferenceCode)
 		return objectFolder.getObjectFolderId();
 	}
 
+	private Page<ObjectDefinition> _search(
```

Comment:
Could you rename it for: _getObjectDefinitionsPage()?
I saw this pattern in ObjectFieldResourceImpl.java.